lib/rubocop/cop/style/hash_conversion.rb

```diff
@@ -18,6 +18,15 @@ module Style
       #
       #   # good
       #   {key1 => value1, key2 => value2}
+      #
+      # @example AllowSplatArgument: true (default)
```

Comment:
Might be a good idea to add some explanation for the need for some such configuration option, as without the context from the ticket I don't think that'd be obvious.

Reply:
Sure! I added an explanation for the configuration option.